src/main.py

```diff
@@ -6,15 +6,18 @@
 from discord.ext import commands
 
 CONFIG = configparser.ConfigParser()
-CONFIG.read("./config.ini")
```

Comment:
I'd send this in as a second PR in the future - although I wouldn't accept it. (Look at #7)